docs/core/tools/project-json.md

```diff
@@ -1,1662 +1,8 @@
-
-For example:
-
-```json
-"frameworks": {
-    "netcoreapp1.0": {
-        "buildOptions": {
-            "define": ["FOO", "BIZ"]
-        }
-    }
-}
-```
-
-### dependencies
-Type: Object
-
-Dependencies that are specific for this framework. This is useful in scenarios where you cannot simply specify a package-level dependency across all targets. Reasons for this can include one target lacking built-in support that other targets have, or requiring a different version of a dependency than other targets. To see a list of the other properties for this node, see the earlier [dependencies](#dependencies) section.
-
-For example:
-
-```json
-    "frameworks": {
-        "netstandard1.5": {
-        "dependencies": {
-            "Microsoft.Extensions.JsonParser.Sources": "1.0.0-rc2-20221"
-        }
-    }
-}
-```
-
-### frameworkAssemblies
-Type: Object
-
-Similar to dependencies but contains reference to assemblies in the GAC that are not NuGet packages. Can also specify the version to use as well as the dependency type. This is used when targeting .NET Framework and Portable Class Library (PCL) targets. You can only build a project with this specified on Windows.
-
-For example:
-
-```json
-"frameworks": {
-    "net451": {
-        "frameworkAssemblies": {
-            "System.Runtime": {
-                "type": "build",
-                "version": "4.0.0"
-            }
-        }
-    }
-}
-```
-
-### wrappedProject
-Type: String
-
-Specifies the location of the dependency project. 
-
-For example:
-
-```json
-"frameworks": {
-    "net451": {
-        "wrappedProject": "MyProject.csproj"
-    }
-}
-```
-
-### bin
-Type: Object
-
-This is used wrap a DLL file. You can reference and generate a package containing this DLL. 
-
-It contains a single String  property, `assembly`, whose value is the assembly path.   
-
-For example:
-
-```json
-"frameworks": {
-    "netcoreapp1.0": {
-        "bin": {
-            "assembly": "c:/otherProject/otherdll.dll"
-        }
-    }
-}
-```
-
-## runtimes
-Type: Object
-
-List of [runtime identifiers (RIDs)](../rid-catalog.md) supported by the project (used when publishing [self-contained deployments](../deploying/index.md#self-contained-deployments-scd)).
-
-For example:
-
-```json
-"runtimes": {
-    "win7-x64": {},
-    "win8-x64": {},
-    "win81-x64": {},
-    "win10-x64": {},
-    "osx.10.11-x64": {},
-    "ubuntu.16.04-x64": {}
-}
-```
-
-## userSecretsId
-Type: String
-
-Specifies a user secret identifier to be used at development-time. 
-For more information, see [Safe storage of app secrets during development](https://docs.microsoft.com/aspnet/core/security/app-secrets).
-
-For example:
-
-```json
-{
-    "userSecretsId": "aspnet-WebApp1-c23d27a4-eb88-4b18-9b77-2a93f3b15119"
-}
-```
+## Content moved!
+This content has merged and moved to the [High-level overview of changes in CLI RC3](../preview3/tools/layering.md) topic. 
```

Comment:
This content is no longer relevant. For information on the changes, see the [High-level overview of changes in CLI RC4](../preview3/tools/layering.md) topic.